stable/search-prod/templates/aggregator-deployment.yaml

```
@@ -114,15 +114,11 @@ spec:
         secret:
           secretName: search-aggregator-secrets
       - name: redis-graph-certs
-        secret:
-          secretName: search-redisgraph-secrets
+        configMap:
+          name: search-redis-ca-crt
           items:
-          - key: ca.crt
-            path: redis.crt
-          - key: tls.crt
-            path: server.crt
-          - key: tls.key
-            path: server.key
+            - key: service-ca.crt
+              path: redis.crt
```

Indentation is different than before, most likely this will cause a yaml parsing error.
```suggestion
          - key: service-ca.crt
            path: redis.crt
```